cmd/influxd/run/server_suite_test.go

```diff
@@ -19,13 +19,13 @@ func init() {
 			&Query{
 				name:    "create database should succeed",
 				command: `CREATE DATABASE db0`,
-				exp:     `{"results":[{}]}`,
+				exp:     `{"results":[{"statement_id":0}]}`,
```

Comment:
It tells the client that the statement was executed. How would you know what the empty response was for without the statement id?